src/goInstallTools.ts

```diff
@@ -134,10 +138,18 @@ function getTools(goVersion: SemVersion): string[] {
 		tools.push('revive');
 	}
 
-	if (goConfig['useLanguageServer']) {
+	if (goConfig['useLanguageServer'] === 'go-langserver') {
 		tools.push('go-langserver');
 	}
 
+	if (goConfig['useLanguageServer'] === 'bingo') {
```

Comment:
bingo is only supported from Go 1.11 onwards, so we should include that check here

Reply:
Ah missed that out, will add the check it in the following patch.

Reply:
Done